src/copyright/ui/HistogramBase.php

```diff
@@ -82,7 +82,10 @@ public function getTableForSingleType($type, $description, $uploadId, $uploadTre
                <div><table border=1 width='100%' id='copyright".$type."'></table></div>
                <br/><br/>
                <div>
-                <a style='cursor: pointer; margin-left:10px;'id='deleteSelected".$type."' class='buttonLink'>Mark selected rows for deletion</a>
+                Replace: <input type='text' id='replaceText'>
```

Comment:
Can it be replaced to following to give user a better editing space?

![picture1](https://user-images.githubusercontent.com/18077542/49781832-2a098e80-fd3a-11e8-882e-89a857227cd0.png)


```suggestion
                Replace: <input type='text' id='replaceText' style='width:80%' />
```